tuist/tuist
# Support appending `arch=x86_64` to destination

Resolves https://github.com/tuist/tuist/issues/5419

### Short description 📝

Adding a new flag `--rosetta` to `tuist build\run\test` for appending `arch=x86_64` to xcodebuild destination.

### How to test the changes locally 🧐

- Create a demo iOS project with tuist and Xcode 14.3 or above.
- Add a `xcframework` dependency which has no support for arm64 simulators.
```
xxx.xcframework
├── ios-arm64_armv7
└── ios-x86_64_i386-simulator
```
- Generate and test your project: `tuist generate && tuist test --rosetta`
- It should work properly.

### Contributor checklist ✅


### Reviewer checklist ✅
